templates/ctl-scaffold/package.json

```diff
@@ -28,7 +28,6 @@
     "@mlabs-haskell/cardano-message-signing": "^1.0.1",
     "@mlabs-haskell/cardano-serialization-lib-gc": "^1.0.10",
     "@mlabs-haskell/json-bigint": "2.0.0",
-    "@mlabs-haskell/uplc-apply-args": "2.0.1",
```

Comment:
Please restore this dependency. We are OK with having unused NPM dependencies, because they will never be included in the bundle

Reply:
Addressed in [4870dac](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1627/commits/4870dac0713220e30e298286666668e34ed20b31)